content/intro-to-storybook/angular/en/composite-component.md

```diff
@@ -226,19 +226,19 @@ As the component grows, so too do input requirements. Define the data requiremen
+In the previous chapter, we learned how to snapshot test stories using Storyshots. With `TaskComponent`, there wasn’t much complexity to test beyond that it renders OK. Since `TaskListComponent` adds another layer of complexity, we want to verify that certain inputs produce certain outputs in a way amenable to automatic testing. To do this, we’ll create unit tests using [Angular Testing Library](https://testing-library.com/docs/angular-testing-library/intro).
 
 ![Testing library logo](/intro-to-storybook/testinglibrary-image.jpeg)
 
 ### Unit tests with Angular Testing Library
 
-Storybook stories paired with manual visual tests and snapshot tests (see above) go a long way to avoiding UI bugs. If stories cover a wide variety of component use cases, and we use tools that ensure a human checks any change to the story, errors are much less likely.
+Storybook stories, manual tests, and snapshot tests go a long way to avoiding UI bugs. If stories cover a wide variety of component use cases, and we use tools that ensure a human checks any change to the story, errors are much less likely.
 
-However, sometimes the devil is in the details. A test framework that is explicit about those details is needed. Which brings us to unit tests.
+However, sometimes the devil is in the details. A test framework that is explicit about those details is needed, bringing us to unit tests.
 
-In our case, we want our `TaskListComponent` to render any pinned tasks **before** unpinned tasks that it is passed in the `tasks` prop. Although we have a story (`WithPinnedTasks`) to test this exact scenario; it can be ambiguous to a human reviewer that if the component **stops** ordering the tasks like this, it is a bug. It certainly won’t scream **“Wrong!”** to the casual eye.
+In our case, we want our `TaskList` to render any pinned tasks **before** unpinned tasks that it has passed in the `tasks` prop. Although we have a story (`WithPinnedTasks`) to test this exact scenario, it can be ambiguous to a human reviewer that if the component **stops** ordering the tasks like this, it is a bug. It certainly won’t scream **“Wrong!”** to the casual eye.
 
-So, to avoid this problem, we can use Angular Testing Library to render the story to the DOM and run some DOM querying code to verify salient features of the output.
+So, to avoid this problem, we can use Angular Testing Library to render the story to the DOM and run some DOM querying code to verify salient features of the output. The nice thing about the story format is that we can import the story in our tests and render it there!
 
 Create a test file called `task-list.component.spec.ts`. Here we’ll build out our tests that make assertions about the output.
```

Comment:
Seems like https://github.com/storybookjs/testing-angular would be a perfect fit here?

Reply:
As far as I can tell, nearly all of the comments left on the Angular pages also apply (with some adjustments) to all frameworks. In this review, I'm not going to repeat these.

Reply:
@kylegach I tried testing angular and it started throwing errors left and right with TypeScript, even following the instructions provided in the Readme, hence why I left the angular testing library

Reply:
Well, that's unfortunate. ☹️  Thanks for attempting! 🙏

Reply:
You know me, no need to thank! I was stoked to get the addon in and start using it but was a bit disappointed when it started to break left and right! I thought it would be an excellent addition to the tutorial.